plugins/publish-changes/README.md

```diff
@@ -0,0 +1,3 @@
+# Strapi plugin publish-changes
+
+A quick description of publish-changes.
```

Comment:
Maybe leave something here for future lost souls (or delete this README)

Reply:
Added some information